cogs/anime/media_list.py

```diff
@@ -151,9 +152,10 @@ async def _manga(self, interaction: Interaction, button: ui.Button[Self]):
 
         if self.manga is None:
             self.manga = await self.fetch_manga()
-            if not self.manga:
-                button.disabled = True
+            if not self.manga["lists"]:
+                self._anime.disabled = True
                 await self.update(interaction)
+                return
```

Comment:
```suggestion
                return await self.update(interaction)
```
This is the style usually used in the bot where applicable.